basis/README.md

```diff
@@ -17,6 +17,16 @@ library.
 [CommandLineProofScript.sml](CommandLineProofScript.sml):
 Proof about the command-line module of the CakeML standard basis library.
 
+[DoubleFFIScript.sml](DoubleFFIScript.sml):
+Specifies behaviour of FFI calls from/toString for doubles
+
+[DoubleProgScript.sml](DoubleProgScript.sml):
```

Comment:
Update this by running `Holmake --no_prereqs README.md` in `basis` after improving the comment at the top of the `DoubleFFIScript.sml` file.